src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -961,9 +960,15 @@ def __init__(
         if max_batch_time is not None:
             self.max_batch_time = max_batch_time
 
+    def shutdown(self) -> None:
+        if self.parallel_executor:
+            self.parallel_executor.shutdown(wait=True)
```

Comment:
@evanpurkhiser why do we need `wait=True`?

Reply:
Actually I will omit it, since it doesn't look like arroyo waits https://github.com/getsentry/arroyo/blob/main/arroyo/processing/strategies/run_task_in_threads.py#L155

And I think, since we're waiting in our `RunTask` arroyo will wait for that to finish anyway before shutting down.